src/message-fragmentation.md

```diff
@@ -43,8 +49,12 @@ Note that:
 - Nodes should enforce a limit on the number of fragments per message (see the [parameters
   chapter](./parameters.md)), by simply discarding fragments with a too-large
   `num_fragments_minus_1`.
+// TODO should common parameter like this be part of the runtime api?
 - Multiple fragments with the same `message_id` and `fragment_index` may be received. It is
   recommended that only one be kept.
+// TODO maybe also acceptable to drop all if they differs in content.
```

Comment:
Hmm yes, though I'm not sure if it's worth adding the code to detect this?

Reply:
having it in spec is free (just mentioning it is ok, even if not implemented).